src/dmclock/cmake/modules/FindGTest.cmake

```diff
@@ -2,27 +2,30 @@
 #
 #  GTEST_INCLUDE_DIRS   - where to find mcas/mcas.h, etc.
 #  GTEST_LIBRARIES      - List of libraries when using mcas.
+#  GTEST_MAIN_LIBRARIES - List of main libraries when using mcas.
```

Comment:
I wasn't aware the cmake versions 2.8.11 and later included these. I will remove them and update the PR.